src/private/WRLDBuildingContour.mm

```diff
@@ -0,0 +1,52 @@
+#import "WRLDBuildingContour.h"
+#import "WRLDBuildingContour+Private.h"
+
+@interface WRLDBuildingContour ()
+
+@end
+
+@implementation WRLDBuildingContour
+{
+    CLLocationDistance m_bottomAltitude;
+    CLLocationDistance m_topAltitude;
+    CLLocationCoordinate2D* m_points;
```

Comment:
raw array of elements + manual element count field - should probably use a container here?
But see earlier comment about possibly modelling using properties backed by C++ std container fields similar to WRLDPolyon.mm